src/sentry/static/sentry/app/utils/componentToString.jsx

```diff
@@ -1,7 +1,5 @@
-import ReactDOM from 'react-dom';
+import ReactDOMServer from 'react-dom/server';
 
 export default function(node) {
```

Comment:
Let's add `componentToString` as a name for this function (there'll be an eslint rule for this).